generate-stackbrew-library.sh

```diff
@@ -114,6 +114,11 @@ for version in "${versions[@]}"; do
 		variantParent="$(awk 'toupper($1) == "FROM" { print $2 }' "$dir/Dockerfile")"
 		variantArches="${parentRepoToArches[$variantParent]}"
 
+    # we don't support the full range of possible archs for these legacy versions...
```

Comment:
Looking at this on GitHub suggest the indentation is totally off.  Otherwise LGTM